konch.py

```diff
@@ -954,6 +955,25 @@ def preview_unauthorized() -> None:
     return None
 
 
+def __ensure_global_cfg_loaded(mod: typing.Union[types.ModuleType, None]) -> None:
+    """Load global config if not already loaded."""
+    try:
+        if (
+            globals()["_cfg"] is None
+            or globals()["_cfg"] == {}
+            or globals()["_cfg"] == Config()
+        ):
+            raise UnboundLocalError
+    except UnboundLocalError:
+        k_atr = "konch"
+        if hasattr(mod, k_atr):
+            k = getattr(mod, k_atr)
```

Comment:
This won't quite work in all cases. For example, the following are not handled

```python
import konch as k
```

```python
from konch import config
```

Reply:
Interesting edge cases, point taken.  If this is expected functionality, perhaps some tests would be helpful.  I will see if I can come up with anything that properly incorporates this scenario.
Thanks again for your feedback.